dashboard/app/helpers/view_options_helper.rb

```diff
@@ -1,32 +1,32 @@
 module ViewOptionsHelper
-  AppViewOptions = Struct.new(*%i(
-    full_width
-    no_header
-    no_footer
-    small_footer
-    code_studio_logo
-    has_i18n
-    white_background
-    callouts
-    autoplay_video
-    channel
-    readonly_workspace
-    is_external_project_level
-    is_channel_backed
-    is_legacy_share
-    post_milestone
-    post_final_milestone
-    puzzle_ratings_url
-    authored_hint_view_requests_url
-    server_level_id
-    game_display_name
-    script_name
-    stage_position
-    level_position
-    public_caching
-    is_13_plus
-    has_contained_levels
-  ))
+  AppViewOptions = Struct.new(
+    :full_width,
+    :no_header,
+    :no_footer,
+    :small_footer,
+    :code_studio_logo,
+    :has_i18n,
+    :white_background,
+    :callouts,
+    :autoplay_video,
+    :channel,
+    :readonly_workspace,
+    :is_external_project_level,
+    :is_channel_backed,
+    :is_legacy_share,
+    :post_milestone,
+    :post_final_milestone,
+    :puzzle_ratings_url,
+    :authored_hint_view_requests_url,
+    :server_level_id,
+    :game_display_name,
+    :script_name,
+    :stage_position,
+    :level_position,
+    :public_caching,
+    :is_13_plus,
+    :has_contained_levels
+  )
```

Comment:
Though I do want to enable the rule, it isn't crystal clear to me that this is an improvement. If you feel strongly, I'm happy to disable the cop for here and below.